server/src/test/java/org/elasticsearch/cluster/ClusterStateDiffIT.java

```diff
@@ -114,13 +115,13 @@ public void testClusterStateDiffSerialization() throws Exception {
                         builder = randomMetaDataChanges(clusterState);
                         break;
                     case 5:
-                        builder = randomVotingConfiguration(clusterState);
+                        builder = randomCoordinationMetaData(clusterState);
                         break;
                     default:
                         throw new IllegalArgumentException("Shouldn't be here");
                 }
             }
-            clusterState = builder.incrementVersion().term(randomLong()).build();
+            clusterState = builder.incrementVersion().build();
```

Comment:
This test asserts that various parts of the coordination metadata are equal. I think it should now just assert that the whole thing is equal.

Reply:
https://github.com/elastic/elasticsearch/pull/35625/commits/04e2324c3948b02eb0321c87b310710c57af95a0